.devcontainer/post_create_command.sh

```
@@ -0,0 +1,29 @@
+source ~/.local/bin/env
+echo "✅ uv installed"
+
+# Install uvicorn (ASGI server)
+echo "🌐 Installing uvicorn..."
+uv tool install uvicorn
+echo "✅ uvicorn installed"
+
+# Install Rust toolchain
+echo "🦀 Installing Rust..."
+curl --proto "=https" --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
+source ~/.cargo/env
```

Shellcheck (0.10.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)